app/views/business_cases/_form.html.haml

```diff
@@ -1,3 +1,14 @@
+%h2 Business case matrix
+.help-block
+  %p The matrix below summarizes three types of annual cash-flows
+  %ol
+    %li <b>Depreciation costs</b>: the investment of technologies and infrastructure divided by their technical lifetime
```

Comment:
Minor gripe, but [go with `strong` instead of `b`](http://stackoverflow.com/a/271776). You can eliminate the HTML by doing:

```haml
%li
  %strong Depreciation costs:
  the investment of technologies and infrastructure divided by their technical lifetime
```